pandas/tools/util.py

```diff
@@ -61,6 +61,13 @@ def to_numeric(arg, errors='raise'):
         - If 'raise', then invalid parsing will raise an exception
         - If 'coerce', then invalid parsing will be set as NaN
         - If 'ignore', then invalid parsing will return the input
+    downcast : numpy.dtype or Python dtype, default None
+        If possible, downcast the resulting numerical data to the
+        specified **numerical** dtype in 'downcast'. If the size of
+        the data's dtype is smaller or equal to that of 'downcast',
```

Comment:
should also take just 'int' (eg strings); just need tests as this should already work